docs/docs/dev_docs/getting_started/sandbox.md

```diff
@@ -206,10 +214,18 @@ Continue with adding the following to the `index.ts` file in our example:
 Running `yarn start` should now output:
 
 ```
-  private-token Aztec Sandbox Info  { version: 1, chainId: 31337 } +0ms
-  private-token Creating accounts using schnorr signers... +2ms
-  private-token Created Alice's account at 0x054d89d0...f17e +23s
-  private-token Created Bob's account at 0x0a8410a1...7c48 +1ms
+  token Aztec Sandbox Info  {
+    version: 1,
+    chainId: 31337,
+    rollupAddress: EthAddress {
+      buffer: <Buffer cf 7e d3 ac ca 5a 46 7e 9e 70 4c 70 3e 8d 87 f6 34 fb 0f c9>
+    },
+    client: 'aztec-rpc@0.1.0',
+    compatibleNargoVersion: '0.11.1-aztec.0'
```

Comment:
Do we want this to be fixed?

Reply:
Not really. Would want it to be depending on the tests, but not sure what the best way is for that right now and just wanted it fixed in this case as people need it shortly